get_number.rb

```diff
@@ -0,0 +1,37 @@
+puts "Welcome to 'Get My Number!"
```

Comment:
You need to use escape characters like [here](https://codingbee.net/ruby/ruby-escaping-character-in-strings)

Reply:
I fixed it